### ISSUE TITLE
[a11y] Screen reader doesn't announce categories before reading tags

### ISSUE THREAD

**Issue by shreyachatterjee00** [civiform/civiform#10120]
To repro: 
Turn screen reader on.
Go to the applicant home page.
Note that the category tags on the program cards aren’t labeled as categories for the  screen reader.  The screen reader just says “Childcare”, out of context.

Expected behavior:
We should add screen-reader-only text that says something like “Childcare category”, or maybe “Categories” before listing all of them.

**Comment by melanie-exygy** [comment]
> Hi [@shreyachatterjee00](https://github.com/shreyachatterjee00) ! I think that we can label the whole group so that it will read out "[Category], unchecked, checkbox, categories, group" for a checkbox item when it first tabs into the group from the beginning or end, but not on subsequent checkboxes. Does that make sense and if so, does that achieve what you're going for? I tried to take a screen recording but can't figure out how to capture the audio

Hey, @tallulahkay, this work item isn't for the category checkboxes.  It's for the category tags, which are on each of the program cards.  

![Image](https://github.com/user-attachments/assets/c2f74800-2490-425f-9ded-acd32856bf48)

Labelling the whole group of tags would be fine.

**Comment by tallulahkay** [comment]
Ope, thanks @melanie-exygy !
